### ISSUE TITLE
U.S. totals implementation

### ISSUE THREAD

**Issue by pekarj** [SuLab/outbreak.info#102]
Would it be possible to implement the features that are "U.S. States only" for the U.S.? Those totals would be helpful for some epidemiologists. Could it not just be a summation of all the data from available states?

**Comment by gkarthik** [comment]
Are you referring to looking at curves for US states individually? We have that implemented [here](https://outbreak.info/epidemiology?location=USA_US-AR%3BUSA_US-CO%3BUSA_US-CT%3BUSA_US-DC%3BUSA_US-FL%3BUSA_US-GA%3BUSA_US-IA%3BUSA_US-IN%3BUSA_US-MA%3BUSA_US-MI%3BUSA_US-NE%3BUSA_US-OH%3BUSA_US-PA%3BUSA_US-PR%3BUSA_US-TX%3BUSA_US-AK%3BUSA_US-AL%3BUSA_US-DE%3BUSA_US-LA%3BUSA_US-MN%3BUSA_US-MT%3BUSA_US-NC%3BUSA_US-NV%3BUSA_US-NY%3BUSA_US-SC%3BUSA_US-UT%3BUSA_US-VT%3BUSA_US-WA%3BUSA_US-WI%3BUSA_GU-X01~%3BUSA_MP-X01~%3BUSA_US-AZ%3BUSA_US-CA%3BUSA_US-HI%3BUSA_US-ID%3BUSA_US-IL%3BUSA_US-KS%3BUSA_US-KY%3BUSA_US-MD%3BUSA_US-ME%3BUSA_US-MO%3BUSA_US-MS%3BUSA_US-ND%3BUSA_US-NH%3BUSA_US-NJ%3BUSA_US-NM%3BUSA_US-OK%3BUSA_US-OR%3BUSA_US-RI%3BUSA_US-SD%3BUSA_US-TN%3BUSA_US-VA%3BUSA_US-WV%3BUSA_US-WY%3BUSA_VI-X01~).

You can search for United States of America (all states/provinces) to get to this view. 
<img width="727" alt="Screen Shot 2020-04-24 at 2 04 47 PM" src="https://user-images.githubusercontent.com/798263/80256915-9d145c00-8634-11ea-84dd-7d40adc8683e.png">

**Comment by pekarj** [comment]
No, I meant for the total US, as opposed to individual states. Is it possible to see all of the US at once for the features that say "US States only"? Not all the state curves on one chart, but the summation of all the curves, or something like that.
